### PR TITLE
Finish 4644: Update the_controller.rst

### PR DESCRIPTION
**Pull request by wouterj**
Replaces https://github.com/symfony/symfony-docs/pull/4644

| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | - |

### CODE REVIEW

File: quick_tour/the_controller.rst [hunk 1]
```diff
@@ -172,8 +172,8 @@ The ``hello`` action will now match URLs like ``/hello/fabien.xml`` or
 like ``/hello/fabien.js``, because the value of the ``_format`` variable doesn't
 meet its requirements.
 
-Redirecting and Forwarding
---------------------------
+Redirecting
```
Comment (xabbuh) [inline quick_tour/the_controller.rst]: We should add a label for the old headline.

Comment (WouterJ) [inline quick_tour/the_controller.rst]: added

**Comment by xabbuh** [comment]
:+1:

**Comment by timglabisch** [comment]
:+1:

**Comment by weaverryan** [comment]
And thanks for finishing this one off too!

**Pull request merged**